src/agent.js

```diff
@@ -1,6 +1,53 @@
 class Agent {
   constructor() {
-    
+    this.travelers = []
```

Comment:
interesting choice to not have any of these properties derive from constructor parameters